GhidraPython/goresym_rename.py

```diff
@@ -0,0 +1,174 @@
+#This script labels Go stripped binaries in Ghidra based on GoReSym output
+#@category Analysis
+
+from ghidra.program.model.symbol.SourceType import USER_DEFINED
+
+import json
+import re
+
+# Maps entry function to the longest _rt0_ function found by GoReSym
+CHOICE_EST_ENTRY_ADDR = "entry to _rt0_ function mapping"
+# Maps pclntab section to TabMeta VA mapping, useful only when pclntab section is present
+CHOICE_EST_PCLNTAB = "pclntab to TabMeta VA mapping"
+# Tries to map addresses against known function names in Ghidra with functions found by GoReSym
+CHOICE_EST_KNOWN_FUNCS = "known function names mapping"
+# All available choices
+CHOICES = [CHOICE_EST_ENTRY_ADDR, CHOICE_EST_PCLNTAB, CHOICE_EST_KNOWN_FUNCS]
```

Comment:
using an [`Enum`](https://docs.python.org/3/library/enum.html) is appropriate here, but only available on py3 :-(

Reply:
Agreed. Now that I have added support for both Python2.7 in Ghidra and Python3 in Ghidratron, I would leave this as-is. I am also using these constants as data for offset strategy choices so I need them anyway.
Usage of `Enum` can be revised later if you feel that this would be a good change to make.